jina/types/arrays/neural_ops.py

```diff
@@ -248,3 +248,27 @@ def visualize(
             plt.savefig(output, bbox_inches='tight', pad_inches=0.1)
         else:
             plt.show()
+
+    @property
+    def embeddings(self, indices: Optional[slice] = None) -> np.ndarray:
```

Comment:
It assumes all of the documents in slice do have `slides`.

And there is no possibility to add arguments to `property`

Reply:
True, updated with a private method to handle slice